datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/corpuser/mappers/CorpUserMapper.java

```diff
@@ -52,9 +56,32 @@ public CorpUser apply(@Nonnull final EntityResponse entityResponse) {
         mappingHelper.mapToResult(CORP_USER_STATUS_ASPECT_NAME,
             (corpUser, dataMap) -> corpUser.setStatus(CorpUserStatusMapper.map(new CorpUserStatus(dataMap))));
         mappingHelper.mapToResult(CORP_USER_CREDENTIALS_ASPECT_NAME, this::mapIsNativeUser);
+
+        mapCorpUserSettings(result, aspectMap.getOrDefault(CORP_USER_SETTINGS_ASPECT_NAME, null));
+
         return mappingHelper.getResult();
     }
 
+    private void mapCorpUserSettings(@Nonnull CorpUser corpUser, EnvelopedAspect envelopedAspect) {
+        CorpUserSettings corpUserSettings = new CorpUserSettings();
+        if (envelopedAspect != null) {
+            corpUserSettings = new CorpUserSettings(envelopedAspect.getValue().data());
+        }
+
+        com.linkedin.datahub.graphql.generated.CorpUserSettings result =
+            new com.linkedin.datahub.graphql.generated.CorpUserSettings();
+
+        AppearanceCorpUserSettings appearanceResult = new AppearanceCorpUserSettings();
```

Comment:
Minor personal nit: I woulda called this CorpUserAppearanceSettings

Reply:
(ie at parity with CorpUserEditableProperties.pdl)

Reply:
This was to keep parity with global settings naming conventions